.github/workflows/reusable-windows-msi.yml

```diff
@@ -24,4 +24,5 @@ jobs:
       with:
         persist-credentials: false
     - name: Build CPython installer
-      run: .\Tools\msi\build.bat --doc -"${ARCH}"
+      run: ./Tools/msi/build.bat --doc -"${ARCH}"
```

Comment:
It seems that with `shell: bash` both `/` and `\\` work for paths, but not `\`.
I've seen (and used) `\\` elsewhere, but since this is the only instance in this file and `/` is more readable we don't need to worry about consistency here.

Reply:
Yes, `/` and `\\` are both valid path separators on Windows, so using `/` always works fine for paths on Windows. The difference between the two platforms is that `/` is not the _default_ path separator on Windows.